content/join.hbs

```diff
@@ -25,17 +25,17 @@
 			</ol>
 			<p class="has-text-centered">This list is not exhaustive; other entrance standards may apply.</p>
 
-			<h1>Join in the <span class="semester-text">Spring 2021</span> semester</h1>
+			<h1>Join in the <span class="semester-text">Fall 2021</span> semester</h1>
 			<div class="columns">
 				<div class="column is-half side-by-side current-semester">
 					<h2>Join as a <span class="semester-text">Georgia Tech</span> student</h1>
-					<p>To enroll for Spring 2021, please first review the <a href="files/Spring 2021 Schedule.pdf">Enrollment Information Packet</a>. Then, download and complete the GT New Student Enrollment (NSE) package below and follow the instructions.</p>
+					<p>To enroll for Fall 2021, please first review the <a href="files/Spring 2021 Schedule.pdf">Enrollment Information Packet</a>. Then, download and complete the GT New Student Enrollment (NSE) package below and follow the instructions.</p>
 					<a class="button is-large" href="files/Spring 2021 NSE GT Application.pdf">NSE Packet for Georgia Tech Students</a>
 				</div>
 				<div class="column is-half side-by-side future-semester">
 					<h2>Join as a <span class="semester-text">crosstown</span> student</h1>
 					<p class="college-list">Detachment 165 hosts students from:<br />Agnes Scott College, Clark Atlanta University, Clayton State University, Emory University, Georgia State University, Kennesaw State University, Morehouse College, Oglethorpe University, Spelman College, Mercer University, and University of West Georgia</p>
-					<p>To enroll for Spring 2021, please first review the <a href="files/Spring 2021 Schedule.pdf">Enrollment Information Packet</a>. Then, download the Crosstown New Student Enrollment (NSE) package, follow the instructions, and satisfy the additional requirements listed below.</p>
+					<p>To enroll for Fall 2021, please first review the <a href="files/Spring 2021 Schedule.pdf">Enrollment Information Packet</a>. Then, download the Crosstown New Student Enrollment (NSE) package, follow the instructions, and satisfy the additional requirements listed below.</p>

```

Comment:
Same for the enrollment packet

Reply:
I asked, but there has not been any response yet. These were the only changes requested to be changed. I will check again